compute_tools/src/informant.rs

```diff
@@ -0,0 +1,50 @@
+use log::{info, warn};
+use std::path::Path;
+use std::process;
+use std::thread;
+use std::time::Duration;
+
+use anyhow::{Context, Result};
+
+static VM_INFORMANT_PATH: &str = "/bin/vm-informant";
```

Comment:
why is it not a `const`?

```rust
const VM_INFORMANT_PATH: &'static str = "/bin/vm-informant";
```

Reply:
Nothing substantial — just habit. With `&'static str` the only difference is that unoptimized builds with `static`s load the length from another static memory location. Unoptimized consts and optimized builds of either inline the length wherever it's used.

Reply:
as I saw, in the codebase, we use `const` more often for that case.
But agree, that's not a big deal

Reply:
Ah ok! I'll change it